.github/workflows/frontend.yml

```diff
@@ -125,6 +126,35 @@ jobs:
           CACHE_TO: type=gha,scope=${{ env.CACHE_SCOPE }}
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
+  scout:
+    runs-on: ubuntu-24.04
+    if: ${{ github.ref == 'refs/heads/master' }}
+    permissions:
+      # required to wirte sarif report
+      security-events: write
+    needs:
+      - image
+    steps:
+      -
+        name: Login to DockerHub
+        uses: docker/login-action@v3
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      -
+        name: Scout
+        id: scout
+        uses: crazy-max/.github/.github/actions/docker-scout@7ea918c968b43fbee013e94ef2dde3e76af5f5a1
```

Comment:
is this one `.github` too many?

Reply:
No it's fine, this is using my account-wide repo `.github`: https://github.com/crazy-max/.github

And action is under: https://github.com/crazy-max/.github/blob/main/.github/actions/docker-scout/action.yml